spec/formatting.md

```diff
@@ -549,6 +549,10 @@ The returned list MAY be empty.
 The most-preferred key is first,
 with each successive key appearing in order by decreasing preference.
 
+If calling MatchSelectorKeys encounters any error,
+a _Bad Selector_ error is emitted
+and an empty list is returned.
```

Comment:
```suggestion
and an empty list is returned.
The fallback representation of such a _message_ is 
the _fallback value_ for the _message_.
```

Reply:
That correction does not match what the algorithm results in. If we get an empty list here, we end up selecting the `*` variant.